common/reviews/api/rush-lib.api.md

```diff
@@ -904,6 +944,14 @@ export class RushUserConfiguration {
     static initializeAsync(): Promise<RushUserConfiguration>;
 }
 
+// Warning: (ae-incompatible-release-tags) The symbol "_setBuildTimes" is marked as @beta, but its signature references "IExecutionResult" which is marked as @alpha
+//
+// @beta (undocumented)
+export function _setBuildTimes(result: IExecutionResult): IBuildTimeRecord[];
```

Comment:
Why does this API need to be exported at all?

Reply:
So that the telemetry logger in `PhasedScriptAction` can get the Build Timings in a formatted way. Would it be better to rewrite the `_setBuildTimes()` function inside of `PhasedScriptAction`?

Reply:
This needs to be exported from the internal module, but doesn't need to be exported from the index file.

Reply:
I see what you mean now. I think most if not all of the functions I exported in the index file can be taken back. I will go through them